vagrant_variables.yml.atomic

```diff
@@ -0,0 +1,34 @@
+---
+
+# DEFINE THE NUMBER OF VMS TO RUN
+mon_vms: 1
+osd_vms: 1
+mds_vms: 0
+rgw_vms: 0
+client_vms: 0
+
+# SUBNET TO USE FOR THE VMS
+subnet: 192.168.0
+
+# MEMORY
+memory: 1024
+
+disks: "[ '/dev/sdb', '/dev/sdc' ]"
+
+eth: 'enp0s3'
+# VAGRANT BOX
```

Comment:
Since the file is atomic oriented you can remove the link of the other boxes I guess ;)